projects/rocr-runtime/runtime/hsa-runtime/hsacore.so.def

```diff
@@ -260,8 +260,10 @@ global:
 	hsa_amd_enable_logging;
 	hsa_amd_signal_wait_all;
 	hsa_amd_portable_export_dmabuf_v2;
-	hsa_amd_ais_file_write;
-	hsa_amd_ais_file_read;
+	hsa_amd_mipmap_array_get_info;
```

Comment:
should be 
  hsa_ext_image_mipmap_array_get_level
  hsa_ext_image_create_v2
  hsa_ext_image_data_get_info_v2
  hsa_ext_image_destroy_v2
The same in runtime/hsa-runtime/hsacore.dll.def

Reply:
These haven't been added in rocr-runtime/runtime/hsa-runtime/hsacore.dll.def, 
hsa_ext_image_mipmap_array_get_level
hsa_ext_image_create_v2
hsa_ext_image_data_get_info_v2
hsa_ext_image_destroy_v2